.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/logs/orderApp-20230711.log
```

Comment:
```suggestion
/logs/*
```
we should go for a more universal solution, let's ignore all the logs in the `logs` folder

Reply:
Done